DeflateWriter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace samdark\sitemap;
+
+/**
+ * Flushes buffer into file with incremental deflating data, available in php 7.0+
```

Comment:
php → PHP

Reply:
I only used this from the original code, but sure ;-)